torch/csrc/jit/codegen/cuda/utils.cpp

```diff
@@ -43,6 +46,12 @@ auto parseDebugDumpOptions() {
         options_map[DebugDumpOption::LaunchParam] = true;
       } else if (token == "segmented_fusion") {
         options_map[DebugDumpOption::FusionSegments] = true;
+      } else if (token == "dump_kernel") {
```

Comment:
I might suggest changing this to `cuda_to_file` since we have `cuda_kernel` and `cuda_full` instead of `dump_kernel` is slightly ambiguous versus the other two.